config.gradle

```diff
@@ -4,17 +4,6 @@ nativeUtils.addWpiNativeUtils()
 nativeUtils.withCrossRoboRIO()
 nativeUtils.withCrossLinuxArm32()
 nativeUtils.withCrossLinuxArm64()
-nativeUtils {
-  wpi {
-    configureDependencies {
```

Comment:
You still need these 3 lines. You just don't need the internal version settings lines. Just leave that block empty.

Reply:
I guess its not _technically_ needed, but I want to keep it just in case.